main.go

```diff
@@ -563,3 +568,56 @@ func deployCloudComponents(plan types.Plan) error {
 
 	return nil
 }
+
+func featureEnabled(features []string, secretFeatures []string) bool {
+	for _, feature := range features {
+		for _, secretFeature := range secretFeatures {
+			if feature == secretFeature {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func filterFeatures(plan types.Plan) types.Plan {
```

Comment:
I see the opportunity for a test table here.

Reply:
Sounds good 👍

Reply:
Tested ✔️